lib/src/js.dart

```diff
@@ -20,6 +24,32 @@ dynamic jsify(obj) {
   return obj;
 }
 
+JsObject uncrackDart2js(dynamic obj) => context.callMethod('uncrackDart2js', [obj]);
```

Comment:
We should add a comment here that this method depends on the embedder defining this method in their JS (`uncrackDart2js`).